third_party/terraform/website/docs/r/google_organization_iam_custom_role.html.markdown

```diff
@@ -60,6 +60,8 @@ exported:
 
 * `deleted` - (Optional) The current deleted state of the role.
 
+* `name` - The name of the role which can be used with iam role bindings in the format `organizations/{{org_id}}/roles/{{role_id}}`
```

Comment:
Let's separate out the definition and make the recommendation a bit more generic. Something like:
```suggestion
* `name` - The name of the role in the format `organizations/{{org_id}}/roles/{{role_id}}`. Like `id`, this field can be used as a reference in other resources such as IAM role bindings.
```